clients/src/main/resources/common/message/FetchRequest.json

```diff
@@ -53,7 +53,9 @@
   //
   // Version 15 adds the ReplicaState which includes new field ReplicaEpoch and the ReplicaId. Also,
   // deprecate the old ReplicaId field and set its default value to -1. (KIP-903)
-  "validVersions": "0-15",
+  //
+  // Version 16 is the same as version 15.
```

Comment:
At the risk of proving my ignorance, why do we bump the version number if nothing has changed? Is it so that the request the same version number as the response (which is bumped)?

Reply:
Yes, from my reading the version of the response is based on the version of the request, so we need to bump both.

Reply:
@chb2ab that's correct.

Reply:
Request and response version should always be the same :)